react/components/Onboarding/components/UI/CTAButton/index.js

```diff
@@ -0,0 +1,11 @@
+import theme from 'react/styles/theme';
+import GenericButton from 'react/components/UI/GenericButton';
+import styled from 'styled-components';
+
+const CTAButton = styled(GenericButton).attrs({
+  f: 6,
+})`
+  margin-top: ${theme.space[6]}
```

Comment:
This can be `mt: 6` in the attrs declaration above.

Also generally you don't want to be accessing the theme directly but rather through the props `${x => x.theme.space[6]}` as this allows the theme to be swapped out dynamically (something we're not doing yet but I can see us doing for things like night-mode, for instance).

Reply:
> This can be mt: 6 in the attrs declaration above.

I looked at `GenericButton` and didnt think it implemented `space` from `styled-system`, which I think is required for `mt`. Ill try this though.

Reply:
Yeah, a bunch of stuff got migrated in a bulk fashion and hasn't been updated yet; working on some of those now.

Reply:
Yeah `mt: 6` in the `attrs` declaration doesnt seem to do anything, I believe because `space` is not yet part of `GenericButton`. `margin-top: ${x => x.theme.space[6]}` works though.

Reply:
👍